basilisk/server_thread.py

```
@@ -125,7 +125,8 @@ def manage_rcv_data(self, data: bytes) -> None:
 			if '\n' in url:
```

Pylint (3.3.7)

[warning] Bad indentation. Found 3 spaces, expected 12

(W0311)